documentation/modules/overview/con-configuration-points-broker.adoc

```diff
@@ -9,57 +9,58 @@ A kafka cluster comprises one or more brokers.
 For producers and consumers to be able to access topics within the brokers, Kafka configuration must define how data is stored in the cluster, and how the data is accessed.
 You can configure a Kafka cluster to run with multiple broker nodes across racks.
 
-Storage::
-Kafka and ZooKeeper store data on disks.
-+
-Strimzi requires block storage provisioned through `StorageClass`.
-The file system format for storage must be _XFS_ or _EXT4_.
```

Comment:
Please do not remove this. This is still valid and important!

Reply:
I moved the second line lower down. 
I removed the `StorageClass` line as it isn't relevant to all the storage types listed (as far as I understand)

Reply:
If we want to return the storageclass line, maybe we should describe it more?  We don't have much on it in the docs. We just say Storage classes define storage profiles and dynamically provision PVs based on that profile.